sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuGenerateExec.scala

```diff
@@ -138,7 +138,8 @@ trait GpuGenerator extends GpuUnevaluable {
   def inputSplitIndices(inputBatch: ColumnarBatch,
       generatorOffset: Int,
       outer: Boolean,
-      targetSizeBytes: Long): Array[Int]
+      targetSizeBytes: Long,
+      maxRows: Int = Int.MaxValue): Array[Int]
```

Comment:
add to javadoc above @param

Reply:
Thanks @tgravescs. Added: https://github.com/NVIDIA/spark-rapids/pull/6631/commits/838203b1ae407c62084180c335ba233722f9eebf.